Tests/SqliteUserModelTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * @copyright  Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE
+ */
+
+namespace Joomla\Database\Tests;
+
+use Joomla\Entity\Tests\Helpers\SqliteCase;
+use Joomla\Entity\Tests\Models\User;
+
+/**
+ *
+ * @since  1.0
+ */
+class SqliteUserModelTest extends SqliteCase
+{
+
+	/**
+	 * @return void
+	 */
+	public function testFind()
+	{
+		$model = new User(self::$driver);
+		$id = $model->findLast()->id;
```

Comment:
You don't need to `findLast` here anymore - you know what the value should be as per the test data (100)